src/assets/i18n/it.po

```diff
@@ -4,10 +4,10 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-msgid " Check Notifications for more details."
+msgid " Controlla le notifiche per maggiori dettagli."
```

Comment:
these wont work, the msgid needs to be the original english and the msgstr needs to be your translation